src/styles/_catalog.scss

```diff
@@ -14,8 +14,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
```

Comment:
add transition just on card class and remove it from hover styles, that way your card will smoothly transformed in both ways (on hover and after hover)